content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -450,6 +450,49 @@ or updating objects that contain Pod templates, such as Deployments, Jobs, State
 See [Enforcing Pod Security at the Namespace Level](/docs/concepts/security/pod-security-admission)
 for more information.
 
+## pod-security.kubernetes.io/exempt
+
+Example: `pod-security.kubernetes.io/exempt: namespace`
+
+Used on: Event
```

Comment:
This isn't used on resources of kind `Event`; in fact, this annotation is not used on resources.
```suggestion
This annotation is not used within the Kubernetes API. When you enable auditing in your
cluster, audit event data is written using `Event` from API group `audit.k8s.io`.
The annotation applies to audit events. Audit events are different from objects in the
[Event API](/docs/reference/kubernetes-api/cluster-resources/event-v1/) (API group
`events.k8s.io`).
```

Reply:
But audit logs aren't composed of kind `Event`? Like this entry of the logs for example?
```json
{
  "kind": "Event",
  "apiVersion": "audit.k8s.io/v1",
  "level": "Metadata",
  "auditID": "c981b4a5-e51a-4c14-b9c1-dabe06fd605a",
  "stage": "ResponseComplete",
  "requestURI": "/api/v1/namespaces/verify-pod-security/pods?fieldManager=kubectl-client-side-apply",
  "verb": "create",
  "user": {
    "username": "minikube-user",
    "groups": [
      "system:masters",
      "system:authenticated"
    ]
  },
  "sourceIPs": [
    "192.168.49.1"
  ],
  "userAgent": "kubectl/v1.23.3 (darwin/amd64) kubernetes/816c97a",
  "objectRef": {
    "resource": "pods",
    "namespace": "verify-pod-security",
    "name": "busybox-privileged",
    "apiVersion": "v1"
  },
  "responseStatus": {
    "metadata": {},
    "status": "Failure",
    "reason": "Forbidden",
    "code": 403
  },
  "requestReceivedTimestamp": "2022-02-02T18:32:40.833098Z",
  "stageTimestamp": "2022-02-02T18:32:40.841531Z",
  "annotations": {
    "authorization.k8s.io/decision": "allow",
    "authorization.k8s.io/reason": "",
    "pod-security.kubernetes.io/audit-violations": "would violate PodSecurity \"restricted:latest\": allowPrivilegeEscalation != false (container \"busybox\" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container \"busybox\" must set securityContext.capabilities.drop=[\"ALL\"]), runAsNonRoot != true (pod or container \"busybox\" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container \"busybox\" must set securityContext.seccompProfile.type to \"RuntimeDefault\" or \"Localhost\")",
    "pod-security.kubernetes.io/enforce-policy": "restricted:latest"
  }
}
```

Reply:
Event from `audit.k8s.io/v1` **!=** Event from `events.k8s.io/v1`.

Unless we're clear about it, some readers will assume that Event means the kind used in the API.

Reply:
There's no HTTP API that lets you create, read, update or delete the `audit.k8s.io/v1` sort of Event.

Reply:
oh 😲 ! All right I see, thanks a lot! It's confusing I admit, I was not very familiar with audits logs and didn't notice they were completely different objects. So what do you think, we should put a disclaimer in front of the three annotations "This annotation is not used within the Kubernetes API" or put them in a separate category then?

Reply:
Both, I think. A separate category _and_ a disclaimer on each audit annotation.

(future work: we can refactor that into a [shortcode](https://gohugo.io/content-management/shortcodes/))

Reply:
Or maybe we could put the whole API version on each specified Kind but it's a bit heavy I suppose? Otherwise, the category would be just the API group? Ok, I can try that!

Reply:
Yes, I could do that! So the idea would be to do the three things in this PR:
1. Separate the page into API group categories.
2. Add a disclaimer on these non `v1` annotations.
3. Add the explicit API group for each Kind of annotation on the page.

And we would have to log the category separation and the addition of the explicit API group feature request on a separate issue? (It's the process that each modification done by a PR should be required by an issue?)

Reply:
We discussed the subject a little bit with sig security docs folks and thought that maybe a good idea would be to publish the missing content first (with the disclaimer you proposed) and open an issue to reorganize this reference section.

As is, it's a bit difficult to reorganize because some annotations lack the kind of resource they are subject to and some are deprecated. So I could open an issue to:
1. Add the missing "Used on" on the annotations.
2. Specify the complete API group for the resources in "Used on".
3. Reorganize the reference page into categories.

Reply:
> We discussed the subject a little bit with sig security docs folks and thought that maybe a good idea would be to publish the missing content first (with the disclaimer you proposed) and open an issue to reorganize this reference section.

:+1:

Reply:
I made the choice to repeat the disclaimer to avoid dependencies between references like [on this one](https://kubernetes.io/docs/reference/labels-annotations-taints/#scheduleralphakubernetesio-preferavoidpods) that adds:
> The taints listed below are always used on Nodes
Which can be skipped if you are only looking for a specific label, annotation or taint without reading the whole document.

I also noticed that the API objects should be in inline code format according to [the style guide](https://kubernetes.io/docs/contribute/style/style-guide/#inline-code-formatting).

Would it be preferable to create multiple issues for the possible independent contributions or create one asking for the whole reorganization/reformatting?

Reply:
> As written, I think this would mislead readers into thinking that audit annotations apply to the [Event](https://github.com/docs/reference/kubernetes-api/cluster-resources/event-v1/) objects - they don't.

I had the same reaction before seeing this comment. This info is good to document, but I don't think it belongs on this page. Putting it on an audit-specific or PodSecurity-specific page seems less misleading

Reply:
We could create a separate page for audit annotations and have the main Well-Known Labels, Annotations and Taints refer to the other page. Maybe that, then?

It'd be nice to put the audit annotation _keys_ somewhere in the main Well-Known Labels, Annotations and Taints  page, I think, so that if people search for any of those, they do find a match.

Reply:
Yes! Originally, the idea would have been to add it "as is", but I admit that it might be confusing... And then to [reorganize the whole reference page](https://github.com/kubernetes/website/issues/31648) to have categories.

Tell me if you think we could organize this differently so I change that PR! :)